waiter/config-full.edn

```diff
@@ -847,13 +847,16 @@
                                      :kind :configured
                                      :configured {:factory-fn 'waiter.token/new-configured-cluster-calculator}
 
-                                     ;; :kind :configured create a calculator that always assigns root as :cluster-config :name
+                                     ;; :kind :regex create a calculator that uses a regex and a map to determine the cluster name
                                      ;; :kind :regex
                                      :regex {:factory-fn 'waiter.token/new-regex-cluster-calculator
                                              ;; A regex to determine the root from the incoming request host header.
-                                             ;; The first group from the match is used to determine the root.
+                                             ;; The first group from the match is used to determine the root using match->cluster.
                                              ;; If no match is found, it defaults to the :cluster-config :name.
-                                             :root-regex #config/regex "^waiter-([a-zA-Z][0-9a-zA-Z\\-]*)"}}
+                                             :host-regex #config/regex "^waiter-([a-zA-Z][0-9a-zA-Z\\-]*)"
+                                             ;; The entry against the first group of the regex match is used to determine the cluster
+                                             :match->cluster {"alias" "name"
+                                                              "name" "name"}}}
```

Comment:
How about if we avoid the regex and just have the mapping use the host portion of the URL? For example:

`waiter-foo.example.com -> foo`
`waiter-bar.example.com -> bar`
`waiter-baz.example.com -> baz`

Reply:
Done.